app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class * (self.clean_power - car.clean_mark)
+                 * self.average_rating / self.distance_from_city_center)
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def serve_cars(self, cars: list) -> float:
+        total_income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def rate_service(self, new_rating: int) -> None:
+        total_rating_sum = self.average_rating * self.count_of_ratings
+        total_rating_sum += new_rating
+        self.count_of_ratings += 1
+        self.average_rating = \
+            round(total_rating_sum / self.count_of_ratings, 1)
```

Comment:
better to use `()` then `\`

Reply:
Can you explain? I'm trying to move the code to the next line, because when I check with pep8 I get an error about the line length being exceeded. The IDE itself substituted "/" during the transfer.

Reply:
Place `round(` on the same line with `=` , `total_rating_sum / self.count_of_ratings, 1` on the next line and `)` on the third line so that closing parenthesis would not have any arguments on the same line as well as an opening parenthesis. In this way you could avoid using line-breaking `\` operator